classroom.rb

```diff
@@ -0,0 +1,13 @@
+class Classroom
+  attr_accessor :label
+
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+
+  def add_student(student)
+    @students.push(student) unless students.include? student
+    student.assign_classroom(self)
+  end
+end
```

Comment:
Nice job so far😉. Since we are reading into the student class, it will be right that we read permission for the student class.  kindly add getters for the student class.

```suggestion
class Classroom
  attr_accessor :label
  attr_reader :students

  def initialize(label)
    @label = label
    @students = []
  end

  def add_student(student)
    @students.push(student) unless students.include? student
    student.assign_classroom(self)
  end
end
```